features/charlie_creates_a_petition.feature

```diff
@@ -46,6 +46,29 @@ Scenario: Charlie creates our petition
   Then a petition should exist with title: "The wombats of wimbledon rock.", state: "validated"
   And there should be a "validated" signature with email "womboid@wimbledon.com" and name "Womboid Wibbledon"
 
+
+Scenario: Charlie creates a petition with valid postcode N1 1TY
+  Given I start a new petition
+  And I fill in the petition details
+  And I press "Next"
+  And I fill in my details with postcode "N1 1TY"
+  And I press "Next"
+  And I fill in sponsor emails
+  And I press "Next"
+  Then I should see my constituency "Islington South and Finsbury"
```

Comment:
Could probably roll this entire scenario into the existing "happy path" scenario.  Maybe.  Anyone else got an opinion?